src/components/CollateralChoice.tsx

```diff
@@ -71,10 +71,8 @@ const CollateralChoice = ({ id }: CollateralChoiceParams) => {
 
   const logoSrc = displayBrandIcon(metrics.totalCollateral.brand);
 
-  const collateralTitle =
-    displayBrandPetname(metrics.totalCollateral.brand).replace('Ibc', '') +
-    '-' +
-    id.substring('manager'.length);
+  // TODO: Come up with a naming scheme after rc0.
```

Comment:
might want to reference https://github.com/Agoric/agoric-sdk/issues/6126

Reply:
This isn't concerning purses but vault managers. We could have multiple vault managers necessitating a better naming scheme while still having a single purse per brand.

Reply:
oops I pasted the wrong link https://github.com/Agoric/agoric-sdk/issues/6518